framework/aster-frame/src/vm/kspace.rs

```diff
@@ -48,7 +48,8 @@ pub(crate) const LINEAR_MAPPING_VADDR_RANGE: Range<Vaddr> =
 /// boot compatibility layer. But we disabled it because the framework
 /// doesn't support relocatable kernel yet.
 pub const fn kernel_loaded_offset() -> usize {
-    0xffff_ffff_8000_0000
+    // 0xffff_ffff_8000_0000
+    0xffff_ffff_0000_0000
```

Comment:
RISC-V's OpenSBI by default will load the kernel to `0x8020_0000`, which conflicts with the original `kernel_loaded_offset`. I also tried to make OpenSBI load the kernel to `0x802_0000`, but it seems don't work.

Reply:
Weird. In x86 we use `0xffff_ffff_8000_0000` as a virtual address which maps to `0x0`. There would be another extra "offset" `0x800_0000` that we load the first segment (`.multiboot_header`). So the loaded kernel would be at `0xffff_ffff_8800_0000` ~ `0xffff_ffff_8fff_ffff`(or somewhere) virtually, and `0x0000_0000_0800_0000` ~ `0x0000_0000_0fff_ffff`(or somewhere).

`0x8020_0000` is about 2G, does SBI want to place it at the end of the physical region?